src/commands/index.ts

```diff
@@ -59,48 +59,68 @@ function toggleAngularPresetCommand() {
   const preset = 'angular';
   const value = getToggleValue(preset);
   const message = `${msg.ngPresetMessage} ${value ? msg.enabled : msg.disabled}. ${msg.restart}`;
-  togglePreset(preset, value, false);
-  showCustomizationMessage(message, [{ title: msg.reload }], applyCustomization, cancel, preset, !value, false);
+  const values = getConfig().inspect(`vsicons.presets.${preset}`);
```

Comment:
I thought of that too but wasn't sure if the flow then was understandable to some. I'm more than willing to refactor that.